src/main/java/io/supertokens/webserver/api/totp/VerifyTotpAPI.java

```diff
@@ -51,6 +52,8 @@ protected void doPost(HttpServletRequest req, HttpServletResponse resp) throws I
         try {
             Totp.verifyCode(main, userId, totp, allowUnverifiedDevices);
 
+            ActiveUsers.updateLastActive(main, userId);
```

Comment:
don't need here since totp is only post login

Reply:
done.